chain/dev/defaults.go

```diff
@@ -28,8 +28,8 @@ var (
 	// DefaultLvl is the default log level
 	DefaultLvl = log.Info
 
-	// DefaultPruningMode is the default pruning mode
-	DefaultPruningMode = "archive"
+	// DefaultPruningEnabled describes if online pruning should be enabled by default.
+	DefaultPruningEnabled = false
```

Comment:
Let me know if you prefer to keep it as a string (might be needed for more pruning options later), but I changed it to a boolean to keep it simpler for now.